src/test/java/org/fundacionjala/coding/Ana/EanTest.java

```diff
@@ -0,0 +1,48 @@
+package org.fundacionjala.coding.Ana;
+
+import org.junit.Before;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+/**
+ * goods have an International Article Number.
+ *
+ * @author Ana Maria Mamani Zenteno.
+ */
+public class EanTest {
+    private Ean ean;
+
+    /**
+     * pruebas antes.
+     *
+     * @throws Exception antes.
+     */
+    @Before
+    public void setUp() {
+        ean = new Ean();
+    }
+
+    /**
+     * checking is true.
+     */
+    @Test
+    public void testEanArticleNumberTrue() {
+        boolean resul = ean.articleNumber("4003301018398");
+        //  boolean resul = ean.articleNumber("0012345678905");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)